### PR TITLE
Union and Insert workouts

### PR DESCRIPTION
**Pull request by mihaiberq**
- [x] Union
- [x] Insert

### CODE REVIEW

**Comment by enki-hq** [comment]
Invalid section title on line 26: ```

**Comment by enki-hq** [comment]
Invalid section title on line 27: --- id | cost | fling_power |

**Comment by mihaiberq** [comment]
@Loopiezlol Why is everything failing?

File: SQL/DQL/insert/insert.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+## Content
+
+The `INSERT` operation allows new entries to be added to the table. The base syntax is as follows:
+```SQL
+INSERT INTO table
+VALUES (value1, value2, value3 ...);
+```
+For example, to insert a new item in the `item` table:
+```SQL
+--- Name is the only text, rest are bigint
```
Comment (lizTheDeveloper) [inline SQL/DQL/insert/insert.md]: Don't show SQL comments, just pull them out of the code sample.

File: SQL/DQL/union/union.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+category: must-know
+
+tags:
+  - introduction
+  - workout
+
+---
+## Content
+
+The **union** of two or more tables means combining their **type-matching rows**. Unlike the `JOIN` operation, which combines columns (containing all entries from either the left, the right or both tables), the `UNION` operation always retrieves every entry in *both tables*.
```
Comment (lizTheDeveloper) [inline SQL/DQL/union/union.md]: This really needs an illustration, and some example results

**Comment by mihaiberq** [comment]
changed base so I can play around with files, will change again after I make the changes.

File: SQL/DQL/union/union.md (hunk 2 above)
Comment (mihaiberq) [reply SQL/DQL/union/union.md]: the example results come later in the insight. I will try to think of an illustration and kindly ask Matt to draw it :+1:

**Comment by enki-hq** [comment]
Invalid section title on line 91: INSERT INTO table

**Pull request merged**